src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/withdrawnappeals/AdminAppealWithdrawnHandler.java

```diff
@@ -41,10 +60,29 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
         if (nonNull(documentDetails)) {
             addToSscsDocuments(caseData, documentDetails);
         }
-
+        cancelHearing(callback);
         return new PreSubmitCallbackResponse<>(caseData);
     }
 
+    private void cancelHearing(Callback<SscsCaseData> callback) {
+        SscsCaseData sscsCaseData = callback.getCaseDetails().getCaseData();
+        log.info("Admin Appeal Withdrawn: Cancel hearing conditions ({}) ({}) ({}) for case ({})",
+                isScheduleListingEnabled, callback.getCaseDetailsBefore().map(CaseDetails::getState)
+                        .orElse(null), sscsCaseData.getSchedulingAndListingFields().getHearingRoute(),
+                                sscsCaseData.getCcdCaseId());
+        if (eligibleForHearingsCancel.test(callback)) {
+            log.info("Admin Appeal Withdrawn: HearingRoute ListAssist Case ({}). Sending cancellation message",
+                    sscsCaseData.getCcdCaseId());
+            hearingMessageHelper.sendListAssistCancelHearingMessage(sscsCaseData.getCcdCaseId(),
+                    CancellationReason.WITHDRAWN);
+        }
+    }
+
+    private final Predicate<Callback<SscsCaseData>> eligibleForHearingsCancel = callback -> isScheduleListingEnabled
+        && SscsUtil.isValidCaseState(callback.getCaseDetailsBefore().map(CaseDetails::getState)
+            .orElse(State.UNKNOWN), List.of(State.HEARING, State.READY_TO_LIST))
+        && SscsUtil.isSAndLCase(callback.getCaseDetails().getCaseData());
```

Comment:
Minor: Is there opportunity to avoid duplication here?  I see another case further up which has very similar code